entrypoint.sh

```diff
@@ -16,6 +16,9 @@ python manage.py pliotenant
 # create default superuser
 python manage.py pliosuperuser
 
+# create default OAuth2 API client credentials
+python manage.py pliooauth2application
```

Comment:
we can avoid using `plio` in this file, right? seems redundant. not only for this command for the command above (`pliosuperuser` too)? might be better to use something that is more meaningful like `createsuperuser` or `createoauth2application`

Reply:
Sure.